bundles/org.openhab.binding.siemensrds/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -145,6 +155,10 @@
 		<label>Thermostat Output State</label>
 		<description>The output state of the the thermostat (Heating, Cooling)</description>
 		<category>fire</category>
+		<tags>
+			<tag>Status</tag>
+			<tag>Heating</tag>
```

Comment:
Based on the description `Mode` might be better, or else?

Reply:
Yup.